I moved the objects to:
[4.30277564 4.30277564]
[4.80277564 7.20277564]
Gathering BC Training data . . .
I moved the objects to:
[4.40277564 3.60277564]
[1.80277564 2.80277564]
Using cache found in C:\Users\tomda/.cache\torch\hub\pytorch_vision_v0.9.0
C:\Users\tomda\anaconda3\Lib\site-packages\torchvision\models\_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
C:\Users\tomda\anaconda3\Lib\site-packages\torchvision\models\_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=ResNet18_Weights.IMAGENET1K_V1`. You can also use `weights=ResNet18_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Traceback (most recent call last):
  File "C:\Users\tomda\Desktop\symmetry_learning\symmetry_shift_runner.py", line 52, in <module>
    main(config)
  File "C:\Users\tomda\Desktop\symmetry_learning\symmetry_shift_runner.py", line 37, in main
    bc_control(config)
  File "C:\Users\tomda\Desktop\symmetry_learning\symmetry_shift_runner.py", line 17, in bc_control
    bcRunner(env)
  File "C:\Users\tomda\Desktop\symmetry_learning\algos\bc.py", line 332, in bcRunner
    bc(env, model)
  File "C:\Users\tomda\Desktop\symmetry_learning\algos\bc.py", line 150, in bc
    trajectories = get_trajectories(env, model, num_trajectories=budget)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\tomda\Desktop\symmetry_learning\algos\bc.py", line 107, in get_trajectories
    plt.show()
  File "C:\Users\tomda\anaconda3\Lib\site-packages\matplotlib\pyplot.py", line 446, in show
    return _get_backend_mod().show(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\tomda\anaconda3\Lib\site-packages\matplotlib\backend_bases.py", line 3620, in show
    cls.mainloop()
  File "C:\Users\tomda\anaconda3\Lib\site-packages\matplotlib\backends\backend_qt.py", line 602, in start_main_loop
    with _maybe_allow_interrupt(qapp):
  File "C:\Users\tomda\anaconda3\Lib\contextlib.py", line 144, in __exit__
    next(self.gen)
  File "C:\Users\tomda\anaconda3\Lib\site-packages\matplotlib\backends\qt_compat.py", line 245, in _maybe_allow_interrupt
    old_sigint_handler(*handler_args)
KeyboardInterrupt